sympy/stats/tests/test_rv.py

```diff
@@ -112,8 +113,8 @@ def test_pspace():
     X, Y = Normal('X', 0, 1), Normal('Y', 0, 1)
     x = Symbol('x')
 
-    raises(ValueError, lambda: pspace(5 + 3))
-    raises(ValueError, lambda: pspace(x < 1))
```

Comment:
Why it used to raise a `ValueError` earlier?

Reply:
When we pass an argument without random expression, it used to raise an error.
while adding a check of `pspace(X) == Pspace()`, this failed in `covariance` when X was a simple symbol. So, I made to return None instead.
Also, to track all the dummy random variables, that were initially having `pspace` as `None`,
are changed to `pspace = PSpace()` instead of keeping them None:
https://github.com/sympy/sympy/blob/94fb720696f5f5d12bad8bc813699fd696afd2fb/sympy/stats/rv.py#L281-L302

Reply:
Well, new changes are theoretically correct, that constant expressions don't have any probability space. Though can we just keep the old thing as it is a change in public interface? Can you try to not make any change here?

Reply:
Okay, I kept it the same